transport/x/grpc/response_writer.go

```diff
@@ -54,7 +53,7 @@ func (r *responseWriter) AddHeaders(headers transport.Headers) {
 }
 
 func (r *responseWriter) SetApplicationError() {
-	r.isApplicationError = true
+	r.AddSystemHeader(ApplicationErrorHeader, "1")
```

Comment:
should we document the "1" somewhere?

Reply:
Changing to error

Reply:
Note that ApplicationErrorHeader documents any non-empty value as indicating error, so this is already implicitly documented.